manifests/stable/supported-nic-ids_v1_configmap.yaml

```diff
@@ -5,6 +5,7 @@ data:
   Intel_i40e_X710_10G: 8086 1572 154c
   Intel_i40e_XL710_40G: 8086 1583 154c
   Intel_i40e_XXV710: 8086 158a 154c
+  Intel_i40e_X710_X557_AT_10G: 8086 1589 154c
```

Comment:
This seems out of place

Reply:
as we spoke offline I slit this change into two different commits